# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/nfs-utils/hotfix-basename.patch.musl
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- nfs-utils-2.8.3/utils/nfsdctl/nfsdctl.c.vanilla	2025-04-09 11:04:22.717369319 +0200
+++ nfs-utils-2.8.3/utils/nfsdctl/nfsdctl.c	2025-04-09 11:04:31.321943614 +0200
@@ -19,6 +19,7 @@
 #include <string.h>
 #include <sched.h>
 #include <sys/queue.h>
+#include <libgen.h>
 
 #include <netlink/genl/family.h>
 #include <netlink/genl/ctrl.h>
